CONTRIBUTING.md

```diff
@@ -63,11 +63,6 @@ Replace "`http://localhost:8001`" with the URL of your running website.
 
 See the [deployment section](README.md#deployment)
 
-## Contribututing
-
-Read [Contributing to IETF Tools projects](https://github.com/ietf-tools/.github/blob/main/CONTRIBUTING.md).
-For this project treat `deploy/preview` brach as `main`.
-
```

Comment:
We should direct any contributors to read https://github.com/ietf-tools/.github/blob/main/CONTRIBUTING.md, especially for the workflow and code/commit styles etc.

Maybe the contributing section should move to the top so people read that first?

> For this project treat `deploy/preview` brach as `main`.
We don't use the `deploy/preview` anymore, so this can go away.

Reply:
> Maybe the contributing section should move to the top so people read that first?

It was originally at the end originally, but I'm good with moving it to the top. I'll remove the line about `deploy/preview` as well.

Reply:
Also, spotted a typo. Fixed now. Ready for review again.